### ISSUE TITLE
dependencies parameter in node template

### ISSUE THREAD

**Issue by zhm** [phated/grunt-jade#18]
I'm trying to use this in a build step of a [component](https://github.com/component/component) to compile jade templates to javascript. Everything is working great, except I have to make one tiny change to the node template to get it to work properly. I know I'm probably abusing the way the node template is intended to work, so feel free to close this and tell me I'm doing it wrong or that this requires a new template type. In the [node template](https://github.com/phated/grunt-jade/blob/master/support/node.template), I have to remove the `./` from the require call. When I remove the `./`, it works perfectly and it's able to find the runtime because of the way component `require` works.

Here is the grunt-jade snippet from my Gruntfile:

``` coffeescript
    jade:
      all:
        files:
          'dist/table/templates': ['lib/table/templates/*.jade']
        options:
          runtime: false
          wrap:
            wrap: true
            amd: false
            node: true
            dependencies: 'jade-runtime'
```

**Comment by zhm** [comment]
For completeness, here is the `node.template` file that works for me within a component.

``` js
var jade = require('<%= dependencies %>');

(function (jade) {
    module.exports = <%= compiledTemplate %>;
})(jade);
```

**Comment by phated** [comment]
I am confused what you are trying to do.

`runtime: false` causes you to not generate the runtime at the same level as the templates and `dependencies: 'jade-runtime'` is supposed to be the name of the runtime that gets generated.

I am assuming you are installing a package called `jade-runtime` in your node_modules and then trying to require it?

**Comment by zhm** [comment]
Sorry for the confusion. I'm not installing `jade-runtime` in `node_modules` because it's being installed and included via component and ran client-side. So the `require` call is being ran in the browser by componentjs, which is why the `./` doesn't work. I'm abusing the fact that the node module pattern mostly works with component.

I made a test app if you're interested in seeing all of the pieces.

https://github.com/zhm/grunt-jade-test

[here](https://github.com/zhm/grunt-jade-test/blob/master/build/build.js#L367) is the actual line from the generated file where `require` is being called from the browser.

**Comment by phated** [comment]
Ok, I see what is going on. Let me know if you think this solution is acceptable:
(not actual code)

``` javascript
if(options.wrap.node){
  dependencies = options.runtime ? './' + options.wrap.dependencies : options.wrap.dependencies;
}
```

and removing the `./` from the template

**Comment by zhm** [comment]
Yep, that looks like it would work. Thanks for looking into this and for making an awesome plugin :)

**Comment by phated** [comment]
Fixed in v0.5.0 and published to npm.
